apps/prairielearn/src/pages/instructorAssessmentInstances/instructorAssessmentInstances.html.ts

```diff
@@ -0,0 +1,583 @@
+                        </h4>
+                      </div>
+                      <div class="modal-body">
+                        Are you sure you want to grade and close all assessment instances for
+                        <strong>
+                          ${resLocals.assessment_set.name} ${resLocals.assessment.number}
+                        </strong>
+                        ? This cannot be undone.
+                      </div>
+                      <div class="modal-footer">
+                        <form name="grade-all-form" method="POST">
+                          <input type="hidden" name="__action" value="close_all" />
+                          <input
+                            type="hidden"
+                            name="__csrf_token"
+                            value="${resLocals.__csrf_token}"
+                          />
+                          <button type="button" class="btn btn-secondary" data-dismiss="modal">
+                            Cancel
+                          </button>
+                          <button type="submit" class="btn btn-primary">Grade and Close all</button>
+                        </form>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex align-items-center">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Students
+              ${resLocals.authz_data.has_course_instance_permission_edit
+                ? html`
+                    <div class="ml-auto">
+                      <div class="dropdown d-flex flex-row">
+                        <button
+                          type="button"
+                          class="btn btn-light dropdown-toggle"
+                          data-toggle="dropdown"
+                          aria-haspopup="true"
+                          aria-expanded="false"
+                        >
+                          Action for all instances <span class="caret"></span>
+                        </button>
+                        <!-- Capture all clicks to dropdown items to prevent scrolling to the top of the page -->
+                        <div
+                          class="dropdown-menu dropdown-menu-right"
+                          onclick="window.event.preventDefault();"
+                        >
+                          ${resLocals.authz_data.has_course_instance_permission_edit
+                            ? html`
+                                <a
+                                  class="dropdown-item"
+                                  role="button"
+                                  tabindex="0"
+                                  data-toggle="modal"
+                                  data-target="#deleteAllAssessmentInstancesModal"
+                                >
```

Comment:
Rather that using `<a role="button" tabindex="0">`, let's just use `<button>`.

Reply:
Updated to use `<button>` in [7efdf17](https://github.com/PrairieLearn/PrairieLearn/pull/9725/commits/7efdf1794fea6c432e21d53c25ae86cac0585ebd).